packages/lesswrong/lib/collections/tags/schema.ts

```diff
@@ -559,6 +559,28 @@ const schema: SchemaType<DbTag> = {
     type: Object,
     foreignKey: 'Tags',
   },
+  
+  autoTagModel: {
+    type: String,
+    label: "Auto-tag classifier model ID",
+    optional: true,
+    viewableBy: ['admins'],
+    editableBy: ['admins'],
+    insertableBy: ['admins'],
+    group: formGroups.advancedOptions,
+    ...schemaDefaultValue(""),
+  },
+  
+  autoTagPrompt: {
```

Comment:
Since these are both new fields and (from a quick skim) none of the scripts/migrations seem to be hydrating existing tags, should these be `nullable`?  Or, if they ought to have a default value of an empty string, we should probably backfill them

Reply:
Sure, marking them nullable. But note that the whole `nullable` flag is not and never has been in a state where you can expect it to be applied reliably (at least for fields of type `String`); there are lots of preexisting fields that are (string, not marked, missing on most object instances).

Reply:
Yep, just want to reduce the future overhead of cleanup 😅